org.sugarlabs.MusicBlocks.json

```diff
@@ -43,6 +43,15 @@
                     "commit": "5a0e5fc244acd8fa806f78a937165cc2a528e68e",
                     "dest": "musicblocks"
                 }
+            ],
+            "cleanup": [
+                "musicblocks/po",
```

Comment:
This won't work, you need to remove (`rm -rf`) these directories in `build-commands` before you copy the files. Also, take a look a `install` command, instead of `cp`.

Reply:
Thank you! I don't think that install will work in this case – unless there's some way to recursively copy the folders? `install -D musicblocks/* app/share/musicblocks` only copies the files outside of directories.